WebJobs.Script.sln

```diff
@@ -294,6 +294,18 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-CSharp-Compat",
 		sample\HttpTrigger-CSharp-Compat\run.csx = sample\HttpTrigger-CSharp-Compat\run.csx
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-CSharp-ClaimsPrincipal", "HttpTrigger-CSharp-ClaimsPrincipal", "{DCBE49B4-CE2C-4D44-8DD9-2B771343E1C6}"
+	ProjectSection(SolutionItems) = preProject
+		sample\HttpTrigger-CSharp-ClaimsPrincipal\function.json = sample\HttpTrigger-CSharp-ClaimsPrincipal\function.json
+		sample\HttpTrigger-CSharp-ClaimsPrincipal\run.csx = sample\HttpTrigger-CSharp-ClaimsPrincipal\run.csx
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Identities", "HttpTrigger-Identities", "{909C3F6B-1B36-498F-8707-C6CC54CD2242}"
+	ProjectSection(SolutionItems) = preProject
+		sample\HttpTrigger-Node-ClaimsPrincipal\function.json = sample\HttpTrigger-Node-ClaimsPrincipal\function.json
```

Comment:
Looks like the naming is off here? You're using two different directory names. Use symmetric naming for this sample for CSharp and Node - e.g. "HttpTrigger-CSharp-Identities" and "HttpTrigger-Identities"?

Reply:
The reason I used different naming is because they are technically slightly different. The CSharp function is using the ClaimsPrincipal binding, while the Node function is using the identities binding data that was provided by the HttpTriggerBindingProvider. While people would likely use them in the same way, but they are different objects. If we prefer consistency, I think renaming HttpTrigger-CSharp-ClaimsPrincipal to HttpTrigger-CSharp-Identities would be alright.

---
In reply to: [216389707](https://github.com/Azure/azure-functions-host/pull/2684#discussion_r216389707) [](ancestors = 216389707)